de/django/README.md

```diff
@@ -1,6 +1,6 @@
 # Django - Was ist das?
 
-Django (*/ˈdʒæŋɡoʊ/ jang-goh*) ist ein freies, quelloffenes Web-Anwendungs-Framework, geschrieben in Python. Ein Web-(Anwendungs-)Framework ist eine Art Baukastensystem, das dir mit vielen vorgefertigten Teilen die Entwicklung von Web-Anwendungen stark erleichtert.
+Django (*/ˈdʒæŋɡoʊ/*) ist ein freies, quelloffenes Web-Anwendungs-Framework, geschrieben in Python. Ein Web-(Anwendungs-)Framework ist eine Art Baukastensystem, das dir mit vielen vorgefertigten Teilen die Entwicklung von Web-Anwendungen stark erleichtert.
```

Comment:
Note: It was a conscious decision to leave out the English-based pronunciation hint in the German translation, and only use the less language-dependent and less ambiguous IPA representation of the pronunciation: #1460